polyfill.js

```diff
@@ -37,5 +37,5 @@ if (typeof Promise.prototype.finally !== 'function') {
 			return newPromise;
 		}
 	};
-	Promise.prototype.finally = shim.finally;
```

Comment:
`Promise` can be polyfilled, but yes, you're right in this case. I'll save the truly rigorous robustness requirements for the [actual polyfill](https://www.npmjs.com/package/promise.prototype.finally).